app/main/views/frameworks.py

```diff
@@ -1124,29 +1124,20 @@ def contract_review(framework_slug, agreement_id):
             agreement_id, current_user.email_address, {'uploaderUserId': current_user.id}
         )
 
-        mandrill_client = DMMandrillClient()
-        email_body = render_template(
-            'emails/framework_agreement_with_framework_version_returned.html',
-            framework_name=framework['name'],
-            framework_slug=framework['slug'],
-        )
-        try:
-            mandrill_client.send_email(
-                to_email_addresses=returned_agreement_email_recipients(supplier_framework),
-                email_body=email_body,
-                subject='Your {} signature page has been received'.format(framework['name']),
-                from_email_address=current_app.config["DM_ENQUIRIES_EMAIL_ADDRESS"],
-                from_name=current_app.config["FRAMEWORK_AGREEMENT_RETURNED_NAME"],
-                tags=['{}-framework-agreement'.format(framework_slug)],
-            )
-        except EmailError as e:
-            current_app.logger.error(
-                "Framework agreement email failed to send. "
-                "error {error} supplier_id {supplier_id} email_hash {email_hash}",
-                extra={'error': str(e),
-                       'supplier_id': current_user.supplier_id,
-                       'email_hash': hash_string(current_user.email_address)})
-            abort(503, "Framework agreement email failed to send")
+        email_client = DMNotifyClient()
+        for email_address in returned_agreement_email_recipients(supplier_framework):
+            try:
+                email_client.send_email(
+                    to_email_address=email_address,
+                    template_name_or_id="framework_agreement_signature_page",
+                    personalisation={
+                        "framework_name": framework['name'],
+                        "framework_slug": framework['slug'],
+                        "framework_updates_url": url_for(".framework_updates", framework_slug=framework["slug"]),
```

Comment:
We need the fully qualified URL here - otherwise the template will be populated with `/suppliers/...`. There should be a utils function to grab this for whatever environment you happen to be in.